src/libraries/System.Net.Http/tests/StressTests/HttpStress/StressClient.cs

```diff
@@ -209,6 +209,11 @@ async Task RunWorker(int taskNum)
                     {
                         _aggregator.RecordCancellation(opIndex, stopwatch.Elapsed);
                     }
+                    catch (Exception e) when (e.Message == "IGNORE")
```

Comment:
I believe - no, because this is a special exception thrown manually to suppress some errors. The throwing side does a version check, so it's redundant here.